### PR TITLE
Update which columns are shown on mobile - Closes #626 

### LINKED ISSUE #626: Main page: Change which column is shown on mobile
### Expected behavior
Latest Transactions: sender, recipient, amount
Latest Blocks: height, transactions, delegate, forged

### Actual behavior
Latest Transactions: id, amount
Latest Blocks: id, height

### PR DESCRIPTION
**Pull request by lsilvs**
### What was the problem?
Important columns being hidden on mobile version

### How did I fix it?
Changed which column is shown when access on mobile

### How to test it?
Access using different screen sizes

### Review checklist
- The PR solves #626 
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices

### CODE REVIEW

**Comment by lsilvs** [comment]
Horizontal scroll bar shows up on Latest Transactions for iPhone 5/SE.
I would use `Txs` rather than `Trs`
`F. LSK` is not clear enough imo

**Comment by MichalTuleja** [comment]
@lsilvs Could you please review the last changes?

File: src/components/blocks/blocks.html [hunk 1]
```diff
@@ -59,8 +56,8 @@ <h1>Blocks <small data-ng-if="vm.blocks.length > 1">{{vm.blocks[0].height}} &#85
                                 <a class="hidden-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator}}</a>
                                 <a class="visible-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator | middleEllipsis:10}}</a>
                             </td>
-                            <td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}}</td>
-                            <td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency}}</td>
+                            <td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}} {{$root.currency.symbol}}</td>
+                            <td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency:1}} {{$root.currency.symbol}}</td>
```
Comment (lsilvs) [inline src/components/blocks/blocks.html]: Using only one decimal place here will lead to 0.0 when BTC is selected as currency (which is wrong).

File: src/components/home/home.html [hunk 2]
```diff
@@ -121,8 +111,8 @@ <h1>Latest Blocks</h1>
 								<a class="hidden-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator}}</a>
 								<a class="visible-xs" href="/address/{{b.generator}}">{{b.delegate.username || b.generator | middleEllipsis:10}}</a>
 							</td>
-							<td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}}</td>
-							<td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency}}</td>
+							<td class="text-right hidden-xs">{{b.totalAmount | currency:$root.currency:$root.decimalPlaces}} {{$root.currency.symbol}}</td>
+							<td class="right-padding-m right-padding-l double text-right">{{b.totalForged | currency:$root.currency:1}} {{$root.currency.symbol}}</td>
```
Comment (lsilvs) [inline src/components/home/home.html]: Same issue when BTC is selected

File: src/components/home/home.html [hunk 3]
```diff
@@ -37,14 +37,7 @@ <h1>Latest Transactions</h1>
...
-							<th class="right-padding-xs right-padding-s right-padding-m text-right">
-								<span class="hidden-xs">
-										Amount ({{$root.currency.symbol}})
-								</span>
-								<span class="visible-xs">
-										{{$root.currency.symbol}}
-								</span>
-							</th>
+							<th class="right-padding-xs right-padding-s right-padding-m text-right">Amount</th>
 							<th class="right-padding-l double text-right hidden-xs hidden-sm hidden-md">Fee ({{$root.currency.symbol}})</th>
```
Comment (lsilvs) [inline src/components/home/home.html]: I would remove the symbol here and add it to the lines to keep the consistency

**Pull request merged**